internal/app/frontend/frontend_service.go

```diff
@@ -354,7 +354,62 @@ func doWatchAssignments(ctx context.Context, id string, sender func(*pb.Assignme
 // AcknowledgeBackfill is used to notify OpenMatch about GameServer connection info.
 // This triggers an assignment process.
 func (s *frontendService) AcknowledgeBackfill(ctx context.Context, req *pb.AcknowledgeBackfillRequest) (*pb.Backfill, error) {
```

Comment:
I see this method also implemented in https://github.com/googleforgames/open-match/pull/1293/ - Is this accidentally in this PR too? or is this PR dependent on the other one and they need to be merged in order?

Reply:
yes, the whole frontend_service.go file was taken from #1293